tests/e2e/performance/csi_tests/test_auto_reclaim_space_multi_clones.py

```diff
@@ -0,0 +1,243 @@
+import logging
+import pytest
+from uuid import uuid4
+
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    performance,
+)
+from ocs_ci.ocs.perftests import PASTest
+from ocs_ci.helpers.performance_lib import run_oc_command
+from ocs_ci.helpers import helpers, performance_lib
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import StorageNotSufficientException, BenchmarkTestFailed
+from ocs_ci.ocs.resources import ocs, pvc
+
+logger = logging.getLogger(__name__)
+ERRMSG = "Error in command"
+
+
+@performance
+@skipif_ocs_version("<4.14")
+class TestReclaimSpaceCronJobMultiClones(PASTest):
+    """
+    This test verifies automatic creation of Reclaim Space Cron Jobs for multiple clones of RBD PVCs in
+    openshift-* namespace. It also verifies that no Reclaim Space Cron Job is created automatically any of the
+    multiple clones of CephFS PVC in this namespace
+    """
+
+    def setup(self):
+        logger.info("Starting the test setup")
+        super(TestReclaimSpaceCronJobMultiClones, self).setup()
+
+        self.num_of_clones = 512
+
+        # Getting the total Storage capacity
+        ceph_capacity = int(self.ceph_cluster.get_ceph_capacity())
+        # Getting the free Storage capacity
+        ceph_free_capacity = int(self.ceph_cluster.get_ceph_free_capacity())
+        # Use 70% of the free storage capacity in the test
+        capacity_to_use = int(ceph_free_capacity * 0.7)
```

Comment:
General Question: Post 70% space utilisation there is still going to be any new IO in the cluster? or all the IO's or creation of new clones will be stopped immediately?